### PR TITLE
T10 - Reverse a Given Number

### PR DESCRIPTION
**Pull request by vikrampg1710-tmm**
T10 - REVERSE A GIVEN NUMBER

Write a program that reverses a given number as explained below and check if it is a palindrome. 

 Input: 948673, Output: 376849 

 Note: The input should first be converted to an integer before the check. Do not do any string manipulations like reversing a string and converting it to an integer.

### CODE REVIEW

File: Assignments/Program.cs [hunk 1]
```diff
@@ -1 +1,43 @@
...
+         Console.WriteLine ($"{ReverseInteger (input)}", Console.ForegroundColor = ConsoleColor.Cyan);
+         Console.ResetColor ();
+      }
+      else Console.WriteLine ("Incorrect Input", Console.ForegroundColor = ConsoleColor.Red);
+      Console.ResetColor ();
+   }
+
+   /// <summary>Returns the reversed integer of a input number</summary>
+   public static int ReverseInteger (int input) {
+      int rInput = 0, sign = 1, d;
```
Comment (asfar-nazar) [inline Assignments/Program.cs:43]: Can this be written in a cleaner way by converting input to a string ?

File: Assignments/Program.cs [hunk 2]
```diff
@@ -1 +1,43 @@
...
+      foreach (int i in testCases) {
+         Console.Write ($"{count++}. {i} ==> ");
+         Console.WriteLine ($"{ReverseInteger (i)}", Console.ForegroundColor = ConsoleColor.Cyan);
+         Console.ResetColor ();
+      }
+      Console.Write ("\nNow, lets try! Please enter an number: ");
+      string num = Console.ReadLine ();
+      if (num.Length < 10 && int.TryParse (num, out int input)) { 
+         Console.Write ($"{input} ==> ");
+         Console.WriteLine ($"{ReverseInteger (input)}", Console.ForegroundColor = ConsoleColor.Cyan);
```
Comment (asfar-nazar) [inline Assignments/Program.cs]: Change

**Review by asfar-nazar — CHANGES_REQUESTED** [review]

File: Assignments/Program.cs (hunk 1 above)
Comment (vikrampg1710-tmm) [reply Assignments/Program.cs:43]: Not to use any string manipulation is the given condition!

Comment (asfar-nazar) [reply Assignments/Program.cs:43]: Acknowledged

**Review by asfar-nazar — APPROVED** [review]

**Pull request closed without merging**